token-client/src/main/java/com/sap/cloud/security/xsuaa/client/OAuth2ServiceException.java

```diff
@@ -12,15 +12,31 @@
 public class OAuth2ServiceException extends IOException {
 
 	private static final long serialVersionUID = 1L;
+	private Integer statusCode = 0;
```

Comment:
I would suggest to rename statusCode to httpStatusCode so that it is 100% clear what kinf status code it is

Reply:
👍